helpers/templates/createGuildTemplate.ts

```diff
@@ -13,42 +11,17 @@ export async function createGuildTemplate(bot: Bot, guildId: bigint, data: Templ
     throw new Error("The description can only be in between 0-120 characters.");
   }
 
-  return await bot.rest.runMethod<DiscordTemplate>(bot.rest, "post", bot.constants.endpoints.GUILD_TEMPLATES(guildId), {
-    code: data.code,
-    name: data.name,
-    description: data.description,
-    usage_count: data.usageCount,
-    creator_id: data.creatorId,
-    creator: data.creator,
-    created_at: data.createdAt,
-    updated_at: data.updatedAt,
-    source_guild_id: data.sourceGuildId,
-    serialized_source_guild: data.serializedSourceGuild,
-    is_dirty: data.isDirty,
-  });
+  return await bot.rest.runMethod<DiscordTemplate>(
+    bot.rest,
+    "post",
+    bot.constants.endpoints.GUILD_TEMPLATES(guildId),
+    data,
```

Comment:
need to be snake cased

Reply:
No it does not?

Reply:
Why not? ur sending this camelcase to discord? discord accepts camel case for this endpoint?

Reply:
Look at the type, there is no camelcase